pyop2/sequential.py

```diff
@@ -71,10 +71,6 @@ class JITModule(host.JITModule):
     %(buffer_decl)s;
     %(buffer_gather)s
     %(kernel_name)s(%(kernel_args)s);
-    %(layout_decl)s;
-    %(layout_loop)s
-        %(layout_assign)s;
-    %(layout_loop_close)s
```

Comment:
so this branch is rebased on ``remove_host_buffer`` and this change reflects the fact that now coffee does padding without the help of PyOP2